src/Nordea/Components/FiveStarRating.elm

```diff
@@ -0,0 +1,69 @@
+module Nordea.Components.FiveStarRating exposing (FiveStarRating(..), init, view)
+
+import Css exposing (alignItems, backgroundColor, borderRadius, borderStyle, center, color, displayFlex, flexDirection, focus, height, justifyContent, none, outline, padding, rem, row, transparent, width)
+import Html.Styled as Styled exposing (Attribute)
+import Html.Styled.Attributes exposing (css)
+import Html.Styled.Events exposing (onMouseEnter)
+import Nordea.Css exposing (gap2)
+import Nordea.Resources.Colors as Colors
+import Nordea.Resources.Icons as Icons
+import Nordea.Themes as Themes
+
+
+type FiveStarRating msg
+    = FiveStarRating (Config msg)
+
+
+type alias Config msg =
+    { setRating : Int -> msg
+    }
+
+
+init : (Int -> msg) -> FiveStarRating msg
+init setRating =
+    FiveStarRating
+        { setRating = setRating
+        }
+
+
+view : List (Attribute msg) -> Int -> FiveStarRating msg -> Styled.Html msg
+view attrs currentHoverRating (FiveStarRating config) =
+    let
+        iconStarStyle =
+            [ css
+                [ width (rem 1.5)
+                , height (rem 1.5)
+                , color Colors.deepBlue
+                ]
+            ]
+
+        starIcon index =
+            if index <= currentHoverRating then
+                Icons.filledStar iconStarStyle
+
+            else
+                Icons.star iconStarStyle
+
+        starIconButton : Int -> Styled.Html msg
+        starIconButton value =
+            Styled.button
+                [ css
+                    [ displayFlex
+                    , alignItems center
+                    , justifyContent center
+                    , backgroundColor transparent
+                    , borderRadius (rem 0.1)
+                    , padding (rem 0.25)
+                    , focus
+                        [ outline none
+                        , borderStyle none
+                        , Css.property "box-shadow" ("0rem 0rem 0rem 0.2rem " ++ Themes.colorVariable Colors.haasBlue)
+                        ]
+                    ]
+                , onMouseEnter (config.setRating value)
```

Comment:
If I understand correctly, this rating component works purely by hovering, and not by clicking. Then there are some minor stuff I think.

First, this component is not accessible. I couldn't set a rating by keyboard. Then if we are fine with that, the stars make the mouse cursor a pointer, giving a very "clickable" vibe. 

I'd enable setting by clicking and improve accessibility and keep the hover for just highlighting, and not for setting. What do you think?

Reply:
Clicking is needed to "submit" the rating, but I didn't see a reason to specify here what onClick does (I tried that but then I have to specify the function type, e.g Int -> msg). Instead this can be specified in the view-function (in the same way we do with nordea buttons). Here is how I want to submut the rating in my code https://github.com/SGFinansAS/FinansFront/pull/8539/files . I think this also works then with enter-key (from an accessibility standpoint)?

Reply:
Hmm it is possible to submit a rating with keyboard, but yes the hover effect doesn't work, I'll work on this !

Reply:
Yes, you can enter and submit if you provide `onClick` from outside, but I doubt you can actually "fill" stars with keyboard by tabbing or with arrow keys. I would model this more like

```
type alias Config msg =
    { selectedRating : Int or Maybe Int
    , onClickRating: Int -> msg
    }
```

If you wanna submit when clicked, then you can do that in the message given from `onClickRating`. In fact, you can add delays there for first showing/setting the value, and then stepping to the next stage. The way we provide `onClick` in the `attrs` has come different to me than the rest of this library.

Reply:
Now in the current case also, one can click in between the stars despite cursor being non-pointer and trigger the `onClick` in the attrs. I think accessibility-wise there are problems but of course if our entire UX department :sweat_smile: has agreed that for FF this is good enough, then it is fine.

https://github.com/user-attachments/assets/3db6de84-a74a-49d4-a667-ac5776759201